tests/unittests/Foundation/WindowsOnly/NSObjectInternalTests.mm

```diff
@@ -0,0 +1,28 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import <TestFramework.h>
+#import <Foundation/Foundation.h>
+
+TEST(NSObject, NSZombie) { // This test will fail with an AV if zombies do not work.
```

Comment:
I don't know if I actually agree with this? There's some confusion currently because it's difficult to distinguish some of the WinObjC-only APIs, but
1) That won't be true in future, more scoped code reviews
2) Pretty much all of the comments are going to be 'foo is a WinObjC-only API'
3) Especially in future scoped CRs, comments like those in 2) are already going to be self-evident

---
In reply to: [72325897](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72325897) [](ancestors = 72325897)